db/schema.rb

```diff
@@ -173,13 +173,13 @@
     t.integer  "primary_payment_method_id"
     t.string   "type",                      limit: 255
     t.integer  "current_cohort_id"
-    t.string   "invitation_token",          limit: 255
```

Comment:
I did a little bit of research and there is this rails issue that seems to be what is happening here.

https://github.com/rails/rails/issues/19001